livekit/src/room/mod.rs

```diff
@@ -740,6 +792,8 @@ impl RoomSession {
 
         // unpublish & republish tracks
         let published_tracks = self.local_participant.tracks();
+
+        // Should I create a new task?
```

Comment:
what's the intent behind creating a new task?

Reply:
The reason is more like: do we want to wait for the track to be republished before finishing the reconnection logic? (Calling the reconnect event)

Reply:
discussed in Slack